src/webextension/locales/en-US/manage.ftl

```diff
@@ -52,14 +52,22 @@ item-details-heading-new = Create a New Entry
 item-details-heading-edit = Edit Entry
 
 item-details-title = Entry Name
+item-details-title-input
+  .placeholder = Primary Bank
 item-details-origin = Website Address
+item-details-origin-input
+  .placeholder = www.example.com
 item-details-username = Username
+item-details-username-input
+  .placeholder = name@example.org
```

Comment:
>Let's use example.com since that's the placeholder origin. That'll be a little more consistent.

You betcha 👍 

FWIW my thinking was to use a different TLD than the user email to communicate they usually won't be the same. eg: facebook.com and devinreams@gmail.com (facebook != gmail) but com and org are subtle, anyway.

Reply:
Yeah, it's just in that uncanny valley right now where it looks like a mistake. We could also use `mozilla.com` as the example domain and `name@example.com` for the example email?